docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3086,11 +3081,34 @@ repeated field must not include duplicates, &ie; members with the same
 `member_id`. The `weight` field is used instead to logically "repeat" the member
 inside the group.
 
-It is explicitly allowed for the same member to be added to be present in
-multiple groups at the same time. If, as a result, an implementation "stores"
-the action id and parameters in the target in multiple locations, the server
-must update all of those locations when a request to modify such a member is
-made.
+It is explicitly allowed for the same member to be present in multiple groups at
+the same time. If, as a result, an implementation "stores" the action id and
+parameters in the target in multiple locations, the server must update all of
+those locations when a request to modify such a member is made.
+
+#### Rules on Setting `max_size` { #sec-max-size-rules}
+
+The valid values for `max_size` depend on the static `max_group_size` included
+in the P4Info message:
+
+* If `max_group_size` is greater than 0, then `max_size` must be greater than 0,
+  and less than or equal to `max_group_size`. We assume that the target can
+  support up to `max_group_size` *weighted* member entries, or the P4Runtime
+  server would have rejected the Forwarding Pipeline Config. If `max_size` is
+  less than `max_group_size`, the server must return `INVALID_ARGUMENT`.
```

Comment:
Should "less than `max_group_size`" be "greater than `max_group_size`" here?

Reply:
Fixed